force-app/main/default/lwc/navToChatterHome/__tests__/navToChatterHome.test.js

```diff
@@ -0,0 +1,34 @@
+import { createElement } from 'lwc';
+import NavToChatterHome from 'c/navToChatterHome';
+import { getNavigateCalledWith } from 'lightning/navigation';
+// this test uses a mocked navigation plugin.
+// see force-app/test/jest-mocks/navigation.js for the mock
+// and see jest.config.js for jest config to use the mock
+
+describe('c-nav-to-chatter-home', () => {
+    it('navigates to chatter home tab', () => {
+        //values to test later
+        const NAV_TYPE = 'standard__namedPage';
+        const NAV_PAGE = 'chatter';
+
+        // boilerplate code to attach lwc element to virtual dom
+        const element = createElement('c-nav-to-chatter-home', {
+            is: NavToChatterHome
+        });
+        document.body.appendChild(element);
+
+        return Promise.resolve().then(() => {
```

Comment:
Add following comment prior to the `Promise.resolve` to be consistent with other test cases.

```
        // Return a promise to wait for any asynchronous DOM updates. Jest
        // will automatically wait for the Promise chain to complete before
        // ending the test and fail the test if the promise rejects.
```